### PR TITLE
Add Glimmer Snowpack post

### PR DESCRIPTION
**Pull request by jaredgalanis**

### CODE REVIEW

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 1]
```diff
@@ -76,14 +77,21 @@ The blog post is a step-by-step explanation with extensive code examples of usin
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Glimmer apps with Snowpack 🔥️⛄](https://dev.to/rajasegar/building-glimmer-apps-with-snowpack-igc)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+There's an awesome new [blog post](https://dev.to/rajasegar/building-glimmer-apps-with-snowpack-igc) from [Rajasegar Chandran @rajasegar](https://github.com/rajasegar) about how to build your Glimmer app with Snowpack 🔥️⛄!
```
Comment (ijlee2) [inline source/2020-12-18-the-ember-times-issue-170.md]:
```suggestion
There's an awesome new [blog post](https://dev.to/rajasegar/building-glimmer-apps-with-snowpack-igc) from [Rajasegar Chandran (@rajasegar)](https://github.com/rajasegar) about how to build your Glimmer app with Snowpack 🔥️⛄!
```

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 2]
```diff
@@ -76,14 +77,21 @@ The blog post is a step-by-step explanation with extensive code examples of usin
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The Glimmer VM runs the components we know and 💜 in Ember, and it is designed to run them super fast 🔥. Snowpack ⛄️ is a new, lightning-fast frontend build tool designed for the modern web. Snowpack leverages JavaScript's native module system [(ESM)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).
```
Comment (ijlee2) [inline source/2020-12-18-the-ember-times-issue-170.md]: I think we can skip mentioning the abbreviation and add the link to "JavaScript's native module system," which may be more descriptive.

```suggestion
The Glimmer VM runs the components we know and 💜 in Ember, and it is designed to run them super fast 🔥. Snowpack ⛄️ is a new, lightning-fast frontend build tool designed for the modern web. Snowpack leverages [JavaScript's native module system](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).
```

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 3]
```diff
@@ -76,14 +77,21 @@ The blog post is a step-by-step explanation with extensive code examples of usin
...
+
+The post explains how to create a Snowpack app, add dependencies such as a Glimmer and how to configure Snowpack.
```
Comment (ijlee2) [inline source/2020-12-18-the-ember-times-issue-170.md]:
```suggestion
The post explains how to create a Snowpack app, add dependencies such as Glimmer, and how to configure Snowpack.
```

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 4]
```diff
@@ -76,14 +77,21 @@ The blog post is a step-by-step explanation with extensive code examples of usin
...
+
+It's an exciting foray into new technologies that are built to optimize our experience as developers and users of web technologies.
+
+But that is not all. Rajasegar has also created a `create-snowpack-app` [template](https://github.com/rajasegar/snowpack-app-template-glimmer) to bootstrap your Glimmer app and use Snowpack. 
+
+There's also a sample Todo list app built using these exciting new technologies if you want to [check it out](https://todolist-glimmer-snowpack.surge.sh/).
```
Comment (ijlee2) [inline source/2020-12-18-the-ember-times-issue-170.md]: (optional) Capitalization may help separate the words list and app.

```suggestion
There's also a sample Todo List app built using these exciting new technologies if you want to [check it out](https://todolist-glimmer-snowpack.surge.sh/).
```

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 5]
```diff
@@ -76,14 +77,21 @@ The blog post is a step-by-step explanation with extensive code examples of usin
...
+
+Still not enough cool frontend bundler technology for you? Well, Rajasegar has written a [follow up post](https://dev.to/rajasegar/building-glimmer-apps-with-rollup-3jcj) about how to use Rollup with a Glimmer app as well!
```
Comment (ijlee2) [inline source/2020-12-18-the-ember-times-issue-170.md]:
```suggestion
Still not enough cool frontend bundler technology for you? Well, Rajasegar has written a follow-up post about [how to use Rollup with a Glimmer app](https://dev.to/rajasegar/building-glimmer-apps-with-rollup-3jcj) as well!
```

**Review by ijlee2 — APPROVED** [review]
I rearranged a couple of link locations. Can you have a look to see if they are okay?

Thanks for covering both Rajasegar's blog posts in one!

**Comment by amyrlam** [comment]
thank you @jaredgalanis and @ijlee2, much appreciated! due to how slow travis is, since this is not to master/main i'm going to merge it in

**Pull request merged**